### ISSUE TITLE
Authentication not working after update to v2.4.5

### ISSUE THREAD

**Issue by tim-breitenstein-it** [DirectoryTree/LdapRecord#278]
**Environment (please complete the following information):**
 - LDAP Server Type: OpenLDAP
 - PHP Version: 7.3
 - LdapRecord: 2.4.5

**Describe the bug:**
- Authentication not working after update to [v2.4.5](https://github.com/DirectoryTree/LdapRecord/releases/tag/v2.4.5)
- Using authentication like: https://ldaprecord.com/docs/core/v2/authentication/#basic-authentication
- Exception: `ldap_bind() expects parameter 1 to be resource, null given`
- Related to: https://github.com/DirectoryTree/LdapRecord/commit/250de2b281c3cd23bdd53bae54b7f55ba310753e

**Solve the bug:**

Change Authentication from:

```
$connection = new \LdapRecord\Connection([
    'hosts' => ['127.0.0.1'],
]);
```
to:
```
$connection = (new \LdapRecord\Connection([
    'hosts' => ['127.0.0.1'],
]))->connect();
```
or:
```
$connection = new \LdapRecord\Connection([
    'hosts' => ['127.0.0.1'],
]);
$connection->connect();
```

**Comment by stevebauman** [comment]
Hi @tim-breitenstein-it!

I've just released [v2.4.6](https://github.com/DirectoryTree/LdapRecord/releases/tag/v2.4.6) with a patch for this. I've tested this on my own LDAP server and installation and it's back up and running. I apologize for the inconvenience this has caused and really appreciate you creating the issue to inform me.

Let me know if you still encounter this issue! 👍

**Comment by tim-breitenstein-it** [comment]
It works again!
